src/applications/pre-need/components/RaceEthnicityReviewField.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export default function RaceEthnicityReviewField(props) {
+  const { formContext, uiSchema, schema, formData } = props;
+  const ObjectField = props.registry.fields.ObjectField;
+
+  if (!formContext.reviewMode) {
+    return <ObjectField {...props} />;
+  }
+
+  const categories = Object.keys(schema.properties).filter(
+    prop => formData[prop],
+  );
+  return (
+    <div>
```

Comment:
This should be set to `<>`... the reason is on the review & submit page, it wraps this view in a `<dl>`, and while you can have a `<div>` immediately inside a `<dl>`, you can only have it one level deep or axe-coconut yells at you.